storage-proofs/src/post/election/circuit.rs

```diff
@@ -137,7 +135,7 @@ impl<'a, H: 'static + Hasher> Circuit<Bls12> for ElectionPoStCircuit<H> {
         }
 
         // pad to a multiple of md arity
-        let arity = PoseidonMDArity::to_usize();
+        let arity = Tree::Arity::to_usize();
```

Comment:
this should not be changed, this is a fixed arity that depends on the number of challenges, not the tree